docarray/index/backends/weaviate.py

```diff
@@ -831,15 +831,22 @@ def find(
             query,
             score_name: Literal["certainty", "distance"] = "certainty",
             score_threshold: Optional[float] = None,
+            search_field: str = '',
         ) -> Any:
             """
             Find k-nearest neighbors of the query.
 
             :param query: query vector for search. Has single axis.
             :param score_name: either `"certainty"` (default) or `"distance"`
             :param score_threshold: the threshold of the score
+            :param search_field: name of the field to search on
             :return: self
             """
+            if search_field != '':
+                raise ValueError(
+                    'Argument search_field is not supported for WeaviateDocumentIndex.\nSet search_field to an empty string to proceed.'
+                )
```

Comment:
then why do we add it ?

Reply:
1. I want to be able to call find(query=query_emb, search_field=self.search_field) for every backend
2. We already do this for a separate `find()` method in weaviate

Reply:
okay I see